include/math/vector2.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <cmath>
+#include <string.h>
```

Comment:
For memcpy

On Mon, Mar 23, 2015 at 5:39 PM, Matt Soucy <notifications@github.com>
wrote:

> In include/math/vector2.h
> <https://github.com/liam-middlebrook/nautical/pull/16#discussion_r26983158>
> :
>
> > @@ -1,6 +1,7 @@
> >  #pragma once
> >
> >  #include <cmath>
> > +#include <string.h>
>
> Why is this in here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/liam-middlebrook/nautical/pull/16/files#r26983158>.
>



-- 

------------------------------------------------------------
----------------------------------
Liam Middlebrook

Email: liammiddlebrook@gmail.com
GitHub: liam-middlebrook <https://github.com/liam-middlebrook>
Phone: 585-698-9579